main.tex

```diff
@@ -63,25 +62,21 @@
 
 
 %%
-Resistance to pyrethroid insecticides is a major concern for malaria vector control, because these are the only compounds approved for use in insecticide-treated bed-nets (ITNs) and are also widely used for indoor residual spraying (IRS). 
+Resistance to pyrethroid insecticides is a major concern for malaria vector control, because these are the only compounds approved for use in insecticide-treated bed-nets (ITNs). 
 %
-Pyrethroids target the voltage-gated sodium channel (VGSC), an essential component of the mosquito nervous system, but mutations in the \emph{Vgsc} gene can disrupt the activity of these insecticides, inducing a ``knock-down resistance'' phenotype. 
+Pyrethroids target the voltage-gated sodium channel (VGSC), an essential component of the mosquito nervous system, but mutations in the VGSC gene (\emph{Vgsc}) can disrupt the activity of these insecticides, inducing a resistance phenotype. 
 %
 Here we use Illumina whole-genome sequence data from phase 1 of the \emph{Anopheles gambiae} 1000 Genomes Project (Ag1000G) to provide a comprehensive account of genetic variation at the \emph{Vgsc} locus in mosquito populations from 8 African countries.
 %
-In addition to three known resistance variants that alter the protein-coding sequence of the \emph{Vgsc} gene, we describe 19 previously unknown non-synonymous variants at appreciable frequency in one or more populations.
+In addition to three known resistance variants, we describe 18 non-synonymous variants at appreciable frequency in one or more populations that are previously unknown in mosquitoes.
 %
-For each variant we predict a resistance phenotype based on genetic evidence for recent selection, patterns of linkage between variants, the position of the variant within the protein structure, and experimental evidence from other species.
+For each variant we predict a resistance phenotype based on genetic evidence for recent selection, then use analyses of haplotype structure to refine our understanding of the origins and spread of these resistance variants between species and geographical locations.
```

Comment:
I think this needs to be broken into two sentences, to emphasise that there are two major things we are contributing here. One is to study functional variation and discuss/predict phenotype. The second is to do an outbreak analysis. I wouldn’t worry too much about abstract length at this stage, as long as we are under ~400 words or so, better to maintain clarity.